edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/config/EdgeHubConfigParser.cs

```diff
@@ -118,7 +119,14 @@ Option<AuthorizationConfig> ParseAuthorizationConfig(BrokerProperties properties
 
         Option<BridgeConfig> ParseBridgeConfig(BrokerProperties properties)
         {
-            return Option.None<BridgeConfig>();
+            IList<string> errors = this.validator.ValidateBridgeConfig(properties.Bridges);
+            if (errors.Count > 0)
+            {
+                string message = string.Join("; ", errors);
+                throw new InvalidOperationException($"Error validating bridge configuration: {message}");
+            }
+
```

Comment:
If it either throws an exception or returns Some, does it need to use Option?

Reply:
There are 3 cases here: 
- settings present (Some)
- settings absent (None)
- validation error (exception)

I've added a null/empty check to return handle none case.